packages/image-block/src/components/ResponsiveImage.tsx

```diff
@@ -0,0 +1,49 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Asset } from '@frontify/app-bridge';
+
+type ResponsiveImageProps = {
+    image: Asset;
+    containerWidth: number;
+    altText?: string;
+    allowWebp?: boolean;
+    quality?: number;
+};
+
+export const ResponsiveImage = ({
```

Comment:
If we can I would add tests now and then it is tested and easily portable to guideline-blocks-settings

Reply:
Yeah, added a bunch of tests 👍